container/Dockerfile

```diff
@@ -14,4 +14,4 @@
 
 FROM nginx:alpine
 
-RUN apk add --no-cache curl=7.64.0-r2
+RUN apk add --no-cache curl
```

Comment:
That will not pass our dockerfile linter ...

Reply:
Well it should not have passed our limiter :( . Can we use the gcloud container and remove this one??

Reply:
The validate script hits the app as nginx and also execs in and runs curl, so leaving this as-is is probably best.  To strike a balance between linting correctness and having to determine the exact version and retest the entire demo every time alpine releases a new version of curl, I am using a fuzzy version match.